src/main.ts

```diff
@@ -88,6 +89,7 @@ const main = async () => {
 
   for (const [outKey, outExport] of Object.entries(outputs)) {
     core.setOutput(outKey, outExport.value);
+    onOutput(outKey + ' found');
```

Comment:
Was this line added as part of the debugging process? I'm not totally able to make sense of it's intent. Forgive me if it should be obvious. 

I'm going to checkout this PR locally and see if I can add a test. I'll take a closer look locally.

Reply:
yeah, probably, it's been a while :)